components/core/test-locust.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo Installing locustio
+python3 -m pip install locustio
+echo Installing Docker
+if [[ "$OSTYPE" == "darwin"* ]];
+then
+    brew install docker
+else
+    sudo apt-get install docker
+fi
+echo Building and running container
+cd ../..
```

Comment:
I guess instead of changing the directory you can give the relative path to the docker-compose.yml file.